recipes/data-management/quantumleap/settings.bat

```diff
@@ -0,0 +1,15 @@
+@ECHO OFF
```

Comment:
is this documented in the readme?

Reply:
Good, I forgot to mention these. We'd need a windows user to test these. Someone from @smartsdkInfotec maybe?